Makefile

```diff
@@ -131,3 +137,8 @@ optimization-fixtures:
 
 	curl "https://api.mapbox.com/optimized-trips/v1/mapbox/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219;13.418555,52.523215?roundtrip=true&distributions=3,1&access_token=$(MAPBOX_ACCESS_TOKEN)" \
 		-o services-optimization/src/test/resources/optimized_trip_distributions.json
+
+clean:
+	./gradlew clean
+	rm -f */dependency-graph-mapbox-libraries.png
```

Comment:
This is not necessary if we bump the version to `0.4.0` because the graphs are included under the `build` folder 👀 https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/issues/41#issuecomment-382651675
> the graph is located under `build/reports/dependency-graph/`

Reply:
Yup true.